src/graphql/hooks.generated.ts

```diff
@@ -97,6 +97,44 @@ export const AppPermissionFragmentDoc = gql`
   code
 }
     `;
+export const AppEventDeliveriesFragmentDoc = gql`
+    fragment AppEventDeliveries on App {
+  webhooks @include(if: $PERMISSION_MANAGE_APPS) {
```

Comment:
these delivers can be fetched ONLY when i have feature flag enabled, not when i have permission to load it. Obviously permission check is necessary, but you need to load it only when it's needed, meaning when feature is active.